packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -224,7 +235,10 @@ function ProjectSelect({
       placeholder={current?.projectName ?? ''}
       open={open}
       loading={loading}
-      onClick={() => setOpen(curr => !curr)}
+      onClick={e => {
```

Comment:
This onClick doesn't have a typed event parameter like the others.

As an aside, you might consider making a wrapper for these repeated onClick handlers. Something like this:

```ts
type ClickHandler = (e: React.MouseEvent) => void

function getOnClickHandler(elementId: NavElement, callback?: ClickHandler): ClickHandler {
  return function(e) {
    onElementClick(elementId)

    if (callback) {
      callback(e)
    }
  }
}
```

Then you can use it in most cases like:
```js
onClick={getOnClickHandler(NavElement.ProjectNavViewAllProjects)}
```
And in cases like this, you can do:
```js
onClick={getOnClickHandler(NavElement.ProjectNavProjectSelectTrigger, () => setOpen(curr => !curr))}
```

Reply:
Spoke off-line, leaving as-is